app/models/state_file_dependent.rb

```diff
@@ -70,4 +70,13 @@ def eligible_for_child_tax_credit
       (ELIGIBLE_CTC.in? dp_str) && (self.first_name.in? dp_str) && (self.last_name.in? dp_str) && (self.ssn.in? dp_str)
     end
   end
+
+  # Could this and the CTC METHOD be more efficient as methods on the intake which yields all the dependents?
+  def eligible_for_eitc
+    dependents = self.intake&.direct_file_data&.parsed_xml&.css('IRS1040ScheduleEIC QualifyingChildInformation')
+    dependents.any? do |dep|
+      dp_str = dep.to_s
+      (self.first_name.in? dp_str) && (self.last_name.in? dp_str) && (self.ssn.in? dp_str)
+    end
+  end
```

Comment:
Helper for parsing EITC dependents drom direct_file_data.

Reply:
I feel the same about this as I do about the methods on the base intake. It seems like we should be relying on the existing column and parsing this info from the direct file data once.